{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"requestUriNotSupported","kind":"identifier"}],"languages":["swift"],"platforms":["iOS"]}]}],"metadata":{"symbolKind":"case","modules":[{"name":"Passage"}],"role":"symbol","externalID":"s:7Passage9OAuth2APIC25ModelError_oauth2CallbackO22requestUriNotSupportedyA2EmF","roleHeading":"Case","title":"OAuth2API.ModelError_oauth2Callback.requestUriNotSupported","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"requestUriNotSupported","kind":"identifier"}]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback\/requestUriNotSupported"},"hierarchy":{"paths":[["doc:\/\/Passage\/documentation\/Passage","doc:\/\/Passage\/documentation\/Passage\/OAuth2API","doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback"]]},"variants":[{"paths":["\/documentation\/passage\/oauth2api\/modelerror_oauth2callback\/requesturinotsupported"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"kind":"symbol","schemaVersion":{"patch":0,"minor":3,"major":0},"references":{"doc://Passage/documentation/Passage/OAuth2API":{"type":"topic","abstract":[],"kind":"symbol","identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API","fragments":[{"kind":"keyword","text":"class"},{"text":" ","kind":"text"},{"text":"OAuth2API","kind":"identifier"}],"url":"\/documentation\/passage\/oauth2api","title":"OAuth2API","role":"symbol","navigatorTitle":[{"text":"OAuth2API","kind":"identifier"}]},"doc://Passage/documentation/Passage":{"url":"\/documentation\/passage","kind":"symbol","abstract":[{"type":"text","text":"Use Passage to add exclusively Passwordless authentication to your iOS Application."}],"role":"collection","title":"Passage","type":"topic","identifier":"doc:\/\/Passage\/documentation\/Passage"},"doc://Passage/documentation/Passage/OAuth2API/ModelError_oauth2Callback":{"fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"ModelError_oauth2Callback"}],"type":"topic","kind":"symbol","identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback","title":"OAuth2API.ModelError_oauth2Callback","abstract":[{"text":"enum for parameter error","type":"text"}],"url":"\/documentation\/passage\/oauth2api\/modelerror_oauth2callback","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"ModelError_oauth2Callback"}]},"doc://Passage/documentation/Passage/OAuth2API/ModelError_oauth2Callback/requestUriNotSupported":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"requestUriNotSupported"}],"identifier":"doc:\/\/Passage\/documentation\/Passage\/OAuth2API\/ModelError_oauth2Callback\/requestUriNotSupported","abstract":[],"url":"\/documentation\/passage\/oauth2api\/modelerror_oauth2callback\/requesturinotsupported","role":"symbol","type":"topic","title":"OAuth2API.ModelError_oauth2Callback.requestUriNotSupported","kind":"symbol"}}}